docs/openapi.yaml

```diff
@@ -0,0 +1,183 @@
+openapi: 3.0.3
+
+info:
+  title: NVA Download API
+  description: >
+    The NVA Download API.
+  termsOfService: https://nva.unit.no/terms/
+  contact:
+    email: nva@unit.no
+  license:
+    name: MIT/NLOD 2.0
+    url: https://link.to.license
+  version: 1.0.0
+
+externalDocs:
+  description: Find out more about NVA
+  url: https://nva.unit.no
+
+paths:
+  /{identifier}/files/{fileIdentifier}:
+    get:
+      summary: Returns an S3 presigned download URL to a publication file (within json response body)
+      x-amazon-apigateway-integration:
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${NvaDownloadPublicationFileFunction.Arn}/invocations
+        httpMethod: POST
+        type: AWS_PROXY
+      security:
+        - CognitoUserPool: [
+          "https://api.nva.unit.no/scopes/frontend",
+          "aws.cognito.signin.user.admin"
+        ]
+      parameters:
+        - in: path
+          name: identifier
+          description: UUID identifier of the Publication.
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - in: path
+          name: fileIdentifier
+          description: UUID identifier of the Publication File.
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        200:
+          $ref: '#/components/responses/200'
```

Comment:
Can you for 200 request instead of using ref for response, make it a ref to a schema with the returned body? Using a genereic 200 response object can cause many errors down the road when we trey to merge the documentations together and several uses "200" for different things. For the error codes this is probably ok.

        200:
          description: OK
          content:
            application/json:
              schema:
                $ref: "#/components/schemas/******"

Reply:
You are right, I didn't think about that.